week-1-ts-ecosystem/w01e01-typescript-basics/src/type.ts

```diff
@@ -0,0 +1,11 @@
+ type UserAddress = {
+    street: string;
+    no: number;
+}
+
+export type User = {
+    name: string;
+    email: string;
+    age: number;
+    address: UserAddress
+}
```

Comment:
Jest spoko - a ja bym poszedł jeszcze dalej. Nazwał to u Ciebie jako plik: `userModel.ts` albo po prostu `User.ts` tak aby model danych był widoczny.